Maple2.Model/Game/ChatSticker.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Runtime.InteropServices;
+
+namespace Maple2.Model.Game;
+
+[StructLayout(LayoutKind.Sequential, Pack = 4, Size = 12)]
+public readonly struct ChatSticker {
+    public int Id { get; init; }
+    public long ExpiryTime { get; init; }
+
+    public ChatSticker(int id, long expiration = long.MaxValue) {
```

Comment:
I generally see 0 used to signify no expiration, is max value actually correct?

Reply:
Yeah, it's max value if there's no expiration.